EAD/EAD-Session2/validation.js

```diff
@@ -0,0 +1,76 @@
+function nameValidation(){
+    var first_name = document.getElementById("first_name");
```

Comment:
get value here as variable name is more significant for value not for that element.